jmx-metrics/src/test/groovy/io/opentelemetry/contrib/jmxmetrics/InstrumentHelperTest.groovy

```diff
@@ -140,13 +140,10 @@ class InstrumentHelperTest extends Specification {
             metric.data.points.eachWithIndex { point, i ->
                 assert point.attributes == Attributes.of(stringKey("labelOne"), "labelOneValue", stringKey("labelTwo"), "${i}".toString())
 
-                if (metricType == SUMMARY) {
+                if (metricType == HISTOGRAM) {
                     assert point.count == 1
                     assert point.sum == value
-                    assert point.percentileValues[0].percentile == 0
```

Comment:
Buckets are set up by the SDK so doesn't seem much point in asserting them. In general, I've loosened some assertions but they could probably be loosened even further to reduce coupling to SDK implementation details